### ISSUE TITLE
Detect dead instanceof-expressions

### ISSUE THREAD

**Issue by markiewb** [markiewb/nb-additional-hints#44]
See https://netbeans.org/bugzilla/show_bug.cgi?id=132511

```
public class NewClass {

    public void SomeMethod(ArrayList foo) {
        if (foo instanceof Map) {
            System.out.println("No compile-time error");
        }
    }
}
```

**Comment by markiewb** [comment]
Fixed in 1.4.0 - get the snapshot to test it

https://github.com/markiewb/nb-additional-hints/releases/tag/v1.4.0.0

**Comment by markiewb** [comment]
Hint will be removed via #70
